surprise-pairwise-from-embedding.R

```diff
@@ -0,0 +1,151 @@
+library(lsa)
+library(tokenizers)
+library(dplyr)
+library(gtools) #pairwise combinations
+#load tsv
+drama_embedding <- read_tsv("YOUR_DIR/hamlet_embeddings.tsv")
+
+#prerocess the data: 
+# - embeddings as numbers
+# - instead of timestamp just count the nth position of a sentence
+drama_embedding <- drama_embedding %>%
+  rowwise() %>%
+  mutate(embedding_num = list(reticulate::py_eval(embeddings)))

```

Comment:
It's confusing to use py_eval just to parse JSON here. Better to use the jsonlite package.

drama_embedding |> rowwise() |> mutate(embedding_num = embedding |> jsonlite::parse_json() |> list())